test-dssspell.sh

```diff
@@ -1,16 +1,33 @@
 #!/usr/bin/env bash
 set -e
 
-[[ "$ETH_RPC_URL" && "$(seth chain)" == "goerli" ]] || { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
+[[ "$(seth chain --rpc-url="$ETH_RPC_URL")" == "goerli" ]] || { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
 
-export DAPP_BUILD_OPTIMIZE=1
+for ARGUMENT in "$@"
+do
+    KEY=$(echo $ARGUMENT | cut -f1 -d=)
+    VALUE=$(echo $ARGUMENT | cut -f2 -d=)
+
+    case "$KEY" in
+            match)      MATCH="$VALUE" ;;
+            optimizer)  OPTIMIZER="$VALUE" ;;     
+            *)   
+    esac
+done
+
+if [[ -z "$OPTIMIZER" ]]; then
+  # Default to running with optimize=1
+  OPTIMIZER=1
+fi
+
+export DAPP_BUILD_OPTIMIZE="$OPTIMIZER"
 export DAPP_BUILD_OPTIMIZE_RUNS=1
 # DssExecLib 0.0.8
-export DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:0x4aad139a88d2dd5e7410b408593208523a3a891d'
+export DAPP_LIBRARIES='lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:0x4aad139a88d2dd5e7410b408593208523a3a891d'
```

Comment:
This space was necessary before due to an issue in dapptools. Might want to leave it the old way for now.

Paging @hexonaut in case there was an open issue on this.

Reply:
i'll put it back since it also matches what we have on mainnet

Reply:
Yeah there is bug in dapps tools that hasn't been resolved afaik.  I'm not sure how forge handles this leading space.